pom.xml

```diff
@@ -52,7 +52,7 @@
         <spring.osgi.core.version>1.2.1</spring.osgi.core.version>
 
         <javax.persistence.version>2.1.0</javax.persistence.version>
-        <exam.version>3.4.0</exam.version>
+        <exam.version>4.2.0</exam.version>
```

Comment:
please merge master into your branch to get this fix. We have moved away from need this explicit reference to pax-exam